compose/README.md

```diff
@@ -15,12 +15,13 @@ git config --global submodule.recurse true
 - Java 11 (should be specified in JAVA_HOME)
 - [macOs/Linux] Android SDK downloaded via `./compose/frameworks/support/jbdeps/android-sdk/downloadAndroidSdk`
 - [Windows] Android SDK downloaded from Android Studio and specified in ANDROID_SDK_ROOT
+- [Optional, macOs] For AppCode IDE, specify environment variable ANDROID_SDK_ROOT=[YOUR_PATH]/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin (in ~/.zshrc)
```

Comment:
Let me check if it requires for IDEA too

Reply:
Can I merge?

Reply:
In IDEA, SDK should be read automatically.

IDEA (or more specifically Android Plugin for IDEA platform, that should also be the same in AppCode) reads SDK from local.properties, and `androidx` project writes SDK into it via SdkHelper.

I am not 100% sure, that we shouldn't specify SDK for AppCode, but if possible, better to not add different instructions for each IDE (or extra instructions that may not needed).

Could you check, what happen, if you:
1. remove Android SDK in the settings (or point to another the location)
2. close AppCode
3. remove local.properties file
4. open AppCode again.

Will Android SDK point to `jbdeps/android-sdk/darwin`?

Reply:
> In IDEA, SDK should be read automatically.
> 
> IDEA (or more specifically Android Plugin for IDEA platform, that should also be the same in AppCode) reads SDK from local.properties, and `androidx` project writes SDK into it via SdkHelper.kt.
> 
> I am not 100% sure, that we shouldn't specify SDK for AppCode, but if possible, better to not add different instructions for each IDE (or extra instructions that may not be needed).
> 
> Could you check, what happen, if you:
> 
>     1. remove Android SDK in the settings (or point to another the location)
> 
>     2. close AppCode
> 
>     3. remove local.properties file
> 
>     4. open AppCode again.
> 
> 
> Will Android SDK point to `jbdeps/android-sdk/darwin`?

On my Mac it works good and successfully import project. 
After it in local.properties automatically adds this lines:
```
sdk.dir=/Users/dim/Desktop/github/JetBrains/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin
cmake.dir=/Users/dim/Desktop/github/JetBrains/compose-jb/compose/frameworks/support/jbdeps/android-sdk/darwin/native-build-tools
```

@pjBooms Could you test this case on your Mac when you have time?